src/shift.jl

```diff
@@ -0,0 +1,18 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    CoordRefSystems.shift(CRS::Type{<:Projected}; lonₒ=0.0u"°", xₒ=0.0u"m", yₒ=0.0u"m")
+
+Shifts the `CRS` with given longitude origin `lonₒ` in degrees, false easting `xₒ`
+and false northing `yₒ` in meters.
+"""
+shift(CRS::Type{<:Projected}; lonₒ=0.0u"°", xₒ=0.0u"m", yₒ=0.0u"m") = Shifted{CRS,lonₒ,xₒ,yₒ}
```

Comment:
Maybe we can call this `ShiftedProjectedCRS` or `ShiftedProjected`?

Reply:
ShiftedCRS would be enough